tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -539,7 +547,6 @@ OperationResult InternalCommandsHandler::executeReadOnlyCommand(uint32_t request
     const uint8_t *request_buffer_as_uint8 = reinterpret_cast<const uint8_t *>(request);
     deserialize(request_buffer_as_uint8, request_buffer_as_uint8 + requestSize, deserialized_request);
   } catch (const runtime_error &e) {
-    outReplySize = 0;
     LOG_ERROR(m_logger, "Failed to deserialize SKVBCRequest: " << e.what());
     return OperationResult::INTERNAL_ERROR;
```

Comment:
The output buffer should be filled here by an error

Reply:
done